cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -55,7 +55,8 @@
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
     <div class="video-player
-                video-player__{{ video_player }}"
+                video-player__{{ video_player }}
+                {{ 'featured-content-module_visual' if is_flexible == false }}"
```

Comment:
I'd just caution that if video-player is on the road to being an organism it shouldn't use `additional_classes`, because then it could arbitrarily change and it can't be guaranteed that its fully covered by documentation, tests, etc.